config/hocon/src/main/java/io/helidon/config/hocon/HoconConfigParser.java

```diff
@@ -111,6 +111,18 @@ public static HoconConfigParser create() {
         return builder().build();
     }
 
+    /**
+     * Create a new instance of HOCON config parser using default configuration.
+     *
+     * @param resolvingEnabled whether to resolve references at the parser level
+     * @return a new instance of parser
+     * @see #builder()
+     */
+    public static HoconConfigParser create(boolean resolvingEnabled) {
```

Comment:
Not sure we should create a shortcut method like this. Is the `resolvingEnabled` really the most commonly used configuration option?

Reply:
Moved to test